webapp-django/crashstats/crashstats/models.py

```diff
@@ -674,10 +674,13 @@ def kwargs_to_params(self, kwargs):
 
         for param in self.get_annotated_params():
             name = param['name']
-            if param['required'] and not kwargs.get(name):
-                raise RequiredParameterError(name)
             value = kwargs.get(name)
-            if not value:
+
+            # 0 is a perfectly fine value, it should not be considered "falsy".
+            if param['required'] and not value and value != 0:
+                raise RequiredParameterError(name)
+
+            if not value and value != 0:
                 continue
```

Comment:
So, if this `continue` happens, does that not mean the value isn't made available? Can't you just do:
```python
if not value and value != 0:
    if param['required']:
        raise RequiredParameterError(name)
    continue
```